homeassistant/components/media_player/__init__.py

```diff
@@ -171,6 +171,8 @@ class MediaPlayerDeviceClass(StrEnum):
     vol.Required(ATTR_MEDIA_CONTENT_ID): cv.string,
     vol.Optional(ATTR_MEDIA_ENQUEUE): cv.boolean,
     vol.Optional(ATTR_MEDIA_EXTRA, default={}): dict,
+    # The frontend stores selector data here. We don't use it in core.
+    vol.Remove("metadata"): dict,
```

Comment:
We already remove "metadata" as part of our `SERVICE_SCHEMA` (`config_validation.py:1082`). 

Should we add this `make_entity_service_schema` to be consistent with that? That way,  `metadata` is always stripped?

Reply:
Sounds good, updated accordingly